cmd/util/latest_version.go

```diff
@@ -0,0 +1,55 @@
+package util
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+const (
+	RepositoryUrl  = "https://github.com/camerondurham/ch"
```

Comment:
These constants could be abstracted into a struct with some implemented methods on it to check latest GH version. Ideally, this latest_version file should be moved into a separate package.